content/en/docs/reference/using-api/client-libraries.md

```diff
@@ -54,7 +54,7 @@ their authors, not the Kubernetes team.
 | DotNet (RestSharp)   | [github.com/masroorhasan/Kubernetes.DotNet](https://github.com/masroorhasan/Kubernetes.DotNet) |
 | Elixir               | [github.com/obmarg/kazan](https://github.com/obmarg/kazan/) |
 | Elixir               | [github.com/coryodaniel/k8s](https://github.com/coryodaniel/k8s) |
-| Go                   | [github.com/ericchiang/k8s](https://github.com/ericchiang/k8s) |
+|||
```

Comment:
We could completely remove the row instead of leaving it empty here.
```suggestion
```

Reply:
thanks for the review. I have applied the changes from my side now.